src/tracer/index.ts

```diff
@@ -0,0 +1,2 @@
+export { AutoblocksTracer } from './tracer';
+export { Evaluation, type TracerEvent, type SendEventArgs } from './models';
```

Comment:
we already export Evaluation from the testing SDK, didn't we discuss keeping all of those exports together?

Reply:
also i don't think these exports are going to have any effect right? only the exports from the entrypoints are reachable